### DISCUSSION TITLE
User authorization using Zitadel API

### DISCUSSION THREAD

**Discussion by gokulraghuthaman** [zitadel/zitadel#7016] [Q&A] [answered]
Hi, I am new to Zitadel and am using the Zitadel API service to create a new user. After creating the new user, how can I add user authorization using the API? I need to create the user with access to add projects.

**Comment by hifabienne** [comment, marked as answer]
hei @gokulraghuthaman 
You can add a membership for the user on the organization level. 
If the user only needs permission to create and manage those projects you can use the role ORG_PROJECT_CREATOR
With that role, the user can create new projects. in the created projects the user will be project owner and therefore be able to manage it.

https://zitadel.com/docs/apis/resources/mgmt/management-service-add-org-member

**Reply by gokulraghuthaman** [reply]
Thank you for your help!
